code/script.js

```diff
@@ -0,0 +1,201 @@
+    
+    // Length of completed projects and sending the number to chart.js
+    const completedNumber = data.filter(item => item.archive_url.includes('project') === true).length
+    completedProjects(completedNumber)
+
+    // Create Header Section
+    header.innerHTML = `
+    <img src="${technigoProjects[0].owner.avatar_url}" class="avatar" alt="profile picture" />
+    <h1>${technigoProjects[0].owner.login}</h1>
+    `
+
+    // For Loop to Create Main Project Elements || & Fetch Number of Commits as well as Pull Requests
+    for (let i = 0; i < technigoProjects.length; i++) {
+
+        // Creating Project classname & project ID
+        let projectName = technigoProjects[i].name.replaceAll("-","");
+        let projectID = technigoProjects[i].id
+
+        // Create DIV in the correct sorted order
+        projects.innerHTML += `
+        <div id="${technigoProjects[i].id}" class="project-wrapper">
+        </div>
+        `
+
+        // Creating the Project Elements
+        document.getElementById(projectID).innerHTML += `
+            <button class="project-header ${projectName}" id="${projectName}" onclick="openSesame('${projectName}')">
+                Week ${i+2}: ${technigoProjects[i].name}
+            </button>
+            <div class="project-info" id="${projectID}2">
+                <p>Repo can be found <a href="${technigoProjects[i].html_url}" target="_blank">here</a>.</p>
+            </div>
+            `
+
+        // Fetching Commit Data
+        commitFetch(technigoProjects[i], projectID);
+
+        // Fetching Pull Requests
+        pullFetch(technigoProjects[i].name, projectID)
+    }
+    }
+
+// Function for Fetching Commits and Live Link
+const commitFetch = (projects, projectsID) => {
+    const GIT_COMMIT_API = projects.commits_url.replace("{/sha}", "")
+    fetch(GIT_COMMIT_API, options)
+        .then(res => res.json())
+        .then(data => {
+
+        // Defining number of commits
+        let numberOfCommits = data.length;
+
+        // Last Commit Message - Committed Netlify Link to add to live link.
+        let lastCommitLink = data[0].commit.message
+
+        // Formatting the DATES
+        // Last Commit Date which is NOT the Netlify commit (Index 1 instead of 0)
+        // Formatting DAYS
+        let lastCommitDateRaw = data[1].commit.committer.date.substring(0,10)
+        // Removing the 0 from dates before the 10th for DAYS
+        let lastCommitDayRaw = lastCommitDateRaw.substring(8,10)
+        let lastCommitDay
+        if (lastCommitDayRaw < 10) {
+            lastCommitDay = lastCommitDayRaw.replace("0", "")
+        } else {
+            lastCommitDay = lastCommitDayRaw;
+        }
+        // Formatting MONTHS
+        const months = ["Jan", "Feb", "Mar", "Apr", "May", "Jun", "Jul", "Aug", "Sep", "Oct", "Nov", "Dec"]
+        let lastCommitMonthNumber = lastCommitDateRaw.substring(5,7).replace("0","")
```

Comment:
Oops - already found a mistake here. When we hit October I'm gonna run into problems 😉